packages/gatsby-theme-docs/package.json

```
@@ -13,6 +13,9 @@
     "url": "https://github.com/commercetools/commercetools-docs-kit.git",
     "directory": "packages/gatsby-theme-docs"
   },
+  "bin": {
+    "create-release-note": "bin/create-release-note.js"
```

Nit: maybe have a more specific script name

```suggestion
    "create-docs-release-note": "bin/create-release-note.js"
```

---

Ok, just don't want the command to be too long :)